nixos/modules/services/hardware/monado.nix

```
@@ -0,0 +1,73 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  inherit (lib.modules) mkDefault mkIf;
+  inherit (lib.options) mkEnableOption mkPackageOption;
```

```suggestion
  inherit (lib) mkEnableOption mkDefault mkIf mkPackageOption;
```
implementation detail which is outside of lib not very important

---

Interesting. I always thought this would be the more precise way to go about it, while `lib.<fn>` is just more convenient. But I guess my way would cause issues if these functions ever get moved